performance/README.md

```diff
@@ -0,0 +1,135 @@
+# neo4j-streams performance test
+
+This part of the project aims to give a standard way to understand the performances of the platform and highlight the configuration results.
+
+
+
+Currently, it's available only the test of neo4j to neo4j configuration, in remote too.
+
+## Setup
+
+The basic unit of work used to test the performances needs a specific cypher part in order to get the timestamp of creation and receiving of the event. The elapsed time is calculated using these timestamps through cypher queries.
+
+### Producer
+
+- neo4j 3.4.7 community edition
+- APOC 3.4.0.3
+- 8 GB heap memory
+- CREATE INDEX ON :Performance(group)
+- default neo4j-streams configuration (without filtering)
+- kafka.batch.size=16384
+
+The  creation query used by the tester:
+
+	UNWIND range(1,{msg}) as ran
+	CREATE (n:Performance)
+	SET n.group = {uuid}, n.creation_time = apoc.date.currentTimestamp()
```

Comment:
why is the uuid passed in from the outside?

Reply:
because the uuid it's only a way to identify in the cunsumer the group of records already created in the producer, generated by the code and passed into the query. The query never changes.